### PR TITLE
Switch prettytable to rich table to support better styles

### PR DESCRIPTION
**Pull request by OwOHamper**
So far only got basics without colors.

- [x] Custom loading bar
- [x] Implement rgb colors
- [x] Style customization

### CODE REVIEW

File: src/table.py [hunk 1]
```diff
@@ -71,17 +101,41 @@ def set_runtime_col_flag(self, field_name: TABLE_COLUMN_NAMES, flag: bool):
...
+        strings = line.split("\x1b[38;2;")
+        del strings[0]
+        for string in strings:
+            splits = string.split("m", 1)
+            rgb = [int(i) for i in splits[0].split(";")]
+            original_strings = splits[1].split("\x1b[0m")
+            string_to_return += f"[rgb({rgb[0]},{rgb[1]},{rgb[2]})]{'[/]'.join(original_strings)}"
+        return string_to_return
+
+    def set_collumns(self):
```
Comment (zayKenyon) [inline src/table.py:132]: Typo: `set_collumns` -> `set_columns`

**Review by zayKenyon — CHANGES_REQUESTED** [review]
Why the comments?

File: src/table.py (hunk 1 above)
Comment (OwOHamper) [reply src/table.py]: fixed

**Comment by OwOHamper** [comment]
> Why the comments?

Temporary comments from old logic gonna keep for little while until it is 100% stable.

**Review by zayKenyon — APPROVED** [review]

**Pull request merged**